app/src/main/java/org/thoughtcrime/securesms/database/GroupDatabase.java

```diff
@@ -458,12 +459,6 @@ public void setActive(String groupId, boolean active) {
     database.update(TABLE_NAME, values, GROUP_ID + " = ?", new String[] {groupId});
   }
 
-  public byte[] allocateGroupId() {
```

Comment:
Seems like there are a few other unused functions in there, might be worth cleaning it while we're doing this.

Reply:
ditto.

```
public byte[] allocateGroupId() {
    byte[] groupId = new byte[16];
    new SecureRandom().nextBytes(groupId);
    return groupId;
  }
  ```